Sources/FoundationEssentials/FileManager/SwiftFileManagerDelegate.swift

```diff
@@ -35,7 +35,7 @@ public protocol FileManagerDelegate : AnyObject, Sendable {
 }
 
 // Default implementations for String-based requirements
-extension FileManagerDelegate {
+package extension FileManagerDelegate {
```

Comment:
I think this should actually be _public_ instead of just package visibility, since we want clients to get these default implementations too (I guess this goes to show how uncommon it is for people to use this delegate on Linux/Windows)

Reply:
Similar here, do you want to make a separate PR for that so we don’t hide functionality change in here?

Reply:
Yeah we can put this in another PR